docker-compose.yml

```diff
@@ -14,6 +44,8 @@ services:
     stdin_open: true
     ports:
       - 9200:9200
+      - 9160:9160
+      - 9140:9140
```

Comment:
these are not strictly needed for LDAP, but good for debugging as they are not encrypted